test/test_dockerpy/README.md

```diff
@@ -2,29 +2,43 @@
 
 Python test suite to validate Podman endpoints using dockerpy library
 
+Validate :-
+
+Make sure podman binary exist under libpod/bin
+
+Validate Python 3.6+ installed using `python --version `
+
+Validate docker-py module exist by using `python list`. If missing install.
+
+
 Running tests
 =============
 To run the tests locally in your sandbox:
 
-#### Run the entire test
+cd into the libpod folder and run one of the following commands:-
+
+Note:- The test can be ran as root or rootless i.e sudo is optional.
+
+#### Run the entire test suite
 
 ```
-sudo PYTHONPATH=/usr/bin/python python -m dockerpy.images
+sudo PYTHONPATH=/usr/bin/python python -m unittest discover test/
```

Comment:
I don't think `PYTHONPATH` does what you think it does. It is a way to extend the default search path for modules; setting it to the executable is not going to do anything.